docs/how-tos/python_consume.md

```diff
@@ -0,0 +1,105 @@
+# Python Consumer
+
+## Preface 
+
+This guide will explain how to fetch data from a GraphQL endpoint and map the response to a 
+Pydantic model. 
+We will cover: 
+
+- Defining a Pydantic model in [Define the Pydantic Model](#define-the-pydantic-model)
+- Fetching the data from GraphQL API in [Fetch Data from GraphQL API](#fetch-data-from-graphql-api)
+- Mapping the response to the Pydantic model in [Map Response Pydantic Model](#map-response-to-pydantic-model)
+
+!!! note "Alternative Options"
+
+    You may want to investigate the following libraries depending on your specific needs:
+      - [`gql`](https://github.com/graphql-python/gql) if you require support for subscriptions
+      - [`sgqlc`](https://github.com/profusion/sgqlc) for typed code generation
+      - [`ariadne-codegen`](https://github.com/mirumee/ariadne-codegen) for `pydantic` compatible code generation
```

Comment:
This doesn't seem to be rendering as a bullet point list, think this should fix it:
```suggestion
    You may want to investigate the following libraries depending on your specific needs:
    
    - [`gql`](https://github.com/graphql-python/gql) if you require support for subscriptions
    - [`sgqlc`](https://github.com/profusion/sgqlc) for typed code generation
    - [`ariadne-codegen`](https://github.com/mirumee/ariadne-codegen) for `pydantic` compatible code generation
```